README.md

```diff
@@ -58,7 +58,7 @@ ably.channels.foo.presence()
 
 ### Credentials
 
-You can provide either a `key` string or `app_id` + `key_id` + `key_value`
+You can provide either a `key` string or `app_id` + `keyId` + `keyValue`
```

Comment:
If we are going going to use camel notation, then should `app_id` not be `appId`?  As an aside, I don't think we needf to support appId, keyId + keyValue anyway as a form of credentials.  We only use the full key now or a token.